subxt/src/blocks/blocks_client.rs

```diff
@@ -39,15 +39,31 @@ where
     T: Config,
     Client: OnlineClientT<T>,
 {
-    /// Obtain block details given the provided block hash, or the latest block if `None` is
-    /// provided.
+    /// Obtain block details given the provided block hash.
     ///
     /// # Warning
     ///
     /// This call only supports blocks produced since the most recent
     /// runtime upgrade. You can attempt to retrieve older blocks,
     /// but may run into errors attempting to work with them.
     pub fn at(
+        &self,
+        block_hash: T::Hash,
+    ) -> impl Future<Output = Result<Block<T, Client>, Error>> + Send + 'static {
+        self.at_optional_block_hash(Some(block_hash))
+    }
+
+    /// Obtain block details of the latest block hash.
+    pub fn at_latest(
+        &self,
+    ) -> impl Future<Output = Result<Block<T, Client>, Error>> + Send + 'static {
+        self.at_optional_block_hash(None)
+    }
+
+    /// Obtain block details given the provided block hash, or the latest block if `None` is
+    /// provided.
+    #[inline]
```

Comment:
@niklasad1 I wanted to keep consistency to how it was before. Before, the pub method was also not inlined. Now the inline will just generate two pub functions `at()` and `at_latest()`that are both not inlined (like before).

Reply:
I see, makes sense but let's hear what @lexnv thinks :)

Reply:
Ah I see; it would have to be async anyway! In that case let';s just remove the `#[inline]` (if anything it's be best on `at()` and `at_latest()` I think) and this looks good to me!